src/main/java/org/openjdk/jextract/impl/Utils.java

```diff
@@ -157,8 +154,7 @@ static boolean isPointer(Type type) {
 
     static boolean isPrimitive(Type type) {
         return switch (type) {
-            case Type.Declared declared when declared.tree().kind() == Declaration.Scoped.Kind.ENUM ->
-                isPrimitive(((Declaration.Constant)declared.tree().members().get(0)).type());
+            case Type.Declared declared when declared.tree().kind() == Declaration.Scoped.Kind.ENUM -> true;
```

Comment:
seems to me that if a type is an enum, it should always be considered a "primitive", no need to recurse